api/rest/v1/bulk_property_values_linked.md

```diff
@@ -0,0 +1,529 @@
+---
+layout: default
+title: Property Values (linked)
+nav_exclude: true
+parent: REST (v1)
+grand_parent: API
+published: true
+permalink: /api/rest/v1/bulk/property/values/in/linked
+---
+
+# /v1/bulk/property/values/in/linked
+
+Get all places of a specific type that are contained in the same parent place, for multiple parent places.
+
+More specifically, this endpoint returns property values for [properties](/glossary.html#property) that can be chained for multiple degrees in the knowledge graph. For example, in the following diagram:
```

Comment:
Maybe put "This endpoint returns property values for ...." as the first sentence. And then claiming "This API for now only supports property `containedInPlace` to fetch `Place` entities and will add support for more properties and entity types"?

Reply:
Done! Leaving the syntax and tables specific to `containedInPlace`. We can update that once the API is generalized to other properties/types.